git_machete/docs.py

```diff
@@ -298,19 +299,50 @@
     "go": """
         <b>Usage: git machete g[o] <direction></b>
         where <direction> is one of: `d[own]`, `f[irst]`, `l[ast]`, `n[ext]`, `p[rev]`, `r[oot]`, `u[p]`
-
```

Comment:
Restore the empty lines for better readability/consistency with the help for other commands